internal/connector/internal/services/connector_cluster.go

```diff
@@ -919,3 +920,20 @@ func (k *connectorClusterService) GetClusterOrg(id string) (string, *errors.Serv
 	}
 	return cluster.OrganisationId, nil
 }
+
+func (k *connectorClusterService) ResetServiceAccount(ctx context.Context, cluster *dbapi.ConnectorCluster) *errors.ServiceError {
+	secret, serr := k.keycloakService.GetKafkaClientSecret(cluster.ClientId)
```

Comment:
I assume this is taking care to cleanup the old secret as well?

Reply:
Yeah, the old secret is reset in SSO service. That's what the incorrectly named `GetKafkaClientSecret()` does. That's why I asked @akoserwal to verify.